src/tribler/gui/event_request_manager.py

```diff
@@ -130,11 +130,10 @@ def on_error(self, error: int, reschedule_on_err: bool):
             return
 
         if self.receiving_data:
-            # We are performing reconnect on the initial connection error only.
-            # In the future, if we consider it useful, we can immediately call here
-            # `self.reconnect(reschedule_on_err=False)`
-            # and raise an exception if it fails to reconnect
-            raise CoreConnectionError('The connection to the Tribler Core was lost')
+            # Most probably Core is crashed. If CoreManager decides to restart the core,
+            # it will also call event_manager.connect_to_core()
+            self._logger.error('The connection to the Tribler Core was lost')
+            return
```

Comment:
Check what is going on when core is killed by `kill`

Reply:
The Tribler GUI correctly shows the error in this case